### DISCUSSION TITLE
Importing hosts

### DISCUSSION THREAD

**Discussion by appcoders** [blinksh/blink#1998] [Support] [answered]
I got a ssh config with about 100 hosts, how can I import them to blink. Thanks.

**Comment by carloscabanero** [comment, marked as answer]
Are they in a .ssh/config file? Just drop it inside .ssh/config in Blink and it will read it from there.

**Reply by carloscabanero** [reply]
Just a heads up, we are working on a way to integrate the hosts from a config file in our UI. Hopefully will be out on 17.3.0
